sys-auth/elogind/elogind-246.10-r2.ebuild

```diff
@@ -51,7 +51,8 @@ DOCS=( README.md src/libelogind/sd-bus/GVARIANT-SERIALIZATION )
 PATCHES=(
 	"${FILESDIR}/${PN}-243.7-nodocs.patch"
 	"${FILESDIR}/${PN}-241.4-broken-test.patch" # bug 699116
-	"${FILESDIR}/${P}-revert-polkit-automagic.patch"
+	"${FILESDIR}/${PN}-246.10-revert-polkit-automagic.patch"
```

Comment:
`${P}` still works here, `${PF}` wouldn't though

Reply:
Addressed the comment, changed back to `${P}`